src/elements/content-picker/__tests__/Footer.test.js

```diff
@@ -25,6 +25,19 @@ describe('elements/content-picker/Footer', () => {
             expect(wrapper.find('.footer-child').length).toBe(1);
         });
 
+        test('should render footer with disabled button', () => {
+            // Wrapper with two nodes
+            const buttons = getWrapper().find('Button');
+
+            // Select the 'Choose' Button
+            const chooseButton = buttons.at(1);
+
+            // https://www.w3.org/WAI/ARIA/apg/patterns/button/
+            // When the action associated with a button is unavailable, the button has aria-disabled set to true.
+            expect(chooseButton.html().includes('aria-disabled')).toBe(true);
```

Comment:
does `prop()` work?

Reply:
it did not, when i printed out props i got only what was explicitly set in Footer
```
    {
      className: 'btn-primary ',
      'aria-label': 'Choose',
      isDisabled: true,
      onClick: [Function: onChoose],
      type: 'button',
      disabled: true,
      children: {
        '$$typeof': Symbol(react.element),
        type: [Function: IconCheck],
        key: null,
        ref: null,
        props: { color: '#fff', height: 16, width: 16 },
        _owner: FiberNode {
          tag: 0,
          key: null,
          elementType: [Function: Footer],
          type: [Function: Footer],
          stateNode: null,
          return: [FiberNode],
          child: [FiberNode],
          sibling: null,
          index: 0,
          ref: null,
          pendingProps: [Object],
          memoizedProps: [Object],
          updateQueue: null,
          memoizedState: null,
          dependencies: null,
          mode: 0,
          flags: 1,
          nextEffect: null,
          firstEffect: [FiberNode],
          lastEffect: [FiberNode],
          lanes: 0,
          childLanes: 1,
          alternate: [FiberNode],
          actualDuration: 0,
          actualStartTime: -1,
          selfBaseDuration: 0,
          treeBaseDuration: 0,
          _debugID: 250,
          _debugSource: null,
          _debugOwner: [FiberNode],
          _debugNeedsRemount: false,
          _debugHookTypes: null
        },
        _store: {}
      },
      isLoading: false,
      showRadar: false
    }

```

i couldn't figure out a way to get at the aria-disabled prop set within button itself